index.js

```diff
@@ -20,11 +20,15 @@ var JSONReporter = function (baseReporterDecorator, config, helper, logger) {
 
   let setFailedExamplesPerFile = function(failure) {
     let title = failure.description
+    let fullTitle;
+
     if (title.includes("hook for")) {
-      const title_regexp = new RegExp(".*hook for (.*)");
+      const title_regexp = new RegExp(".*hook for \"(.*)\"");
```

Comment:
Why the extra quotes are needed now?

Reply:
You are correct, this change is irrelevant with the current fix, it is something that slipped the first time I added this piece of code. The full title ends up with escaped quotes. It is harmless but it doesn't read very well in the console output:
`parent context title blablah .* \"example name\""`